Plutarch/Evaluate.hs

```diff
@@ -42,4 +53,18 @@ evaluateScript s = do
   term' <- runExceptT @FreeVariableError (deBruijnTerm named)
   let Right term = term'
   let s' = Script $ Program () (defaultVersion ()) $ termMapNames unNameDeBruijn term
-  pure (budget, logOut, s')
+  pure (usedBudget, logOut, s')
+
+{- | Evaluate a program in the CEK machine against the given budget, with the
+ usual text dynamic builtins and tracing, additionally returning the trace
+ output.
+-}
+evaluateCekBudgetedTrace ::
+  (uni ~ PLC.DefaultUni, fun ~ PLC.DefaultFun) =>
```

Comment:
These constraints are unnecessary. Just replace `uni` with the relevant types.

Reply:
Done.